bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
 - shell: git rev-parse --show-toplevel 2>/dev/null
   register: toplevel
 
+- name: Ensure that the apt cache is up to date
```

Comment:
This repository generally doesn't make use of `name:`, and in previous reviews @prakashsurya and I both expressed our opinion that we shouldn't be using `name:` in new tasks, for a few reasons:

- Existing tasks don't use `name:`, and we would like to be consistent within the same repository.
- The names were only ever truly useful in order to help people map an error in a production run to a specific source file and line number; however, in the latest version of Ansible, source files and line numbers can be printed for tasks without having to use `name:`.

As such, I would recommend removing the use of `name:` both here and elsewhere.